python/django/transit_indicators/i18n/vi/LC_MESSAGES/django.po

```diff
@@ -654,176 +654,193 @@ msgstr "Lỗi khi tính toán chỉ dẫn"
 msgid "Dwell Time Performance"
 msgstr "Thời gian dừng thực hiện"
 
-#: transit_indicators/models.py:402
+#: transit_indicators/models.py:418
 msgid "Weekly number of hours of service"
 msgstr "Số giờ hoạt động trong tuần"
 
-#: transit_indicators/models.py:403
+#: transit_indicators/models.py:419
 msgid "Job accessibility"
 msgstr "Khả năng đi đến"
 
-#: transit_indicators/models.py:404
+#: transit_indicators/models.py:420
 msgid "Transit system length"
 msgstr "Chiều dài hệ thống giao thông"
 
-#: transit_indicators/models.py:405
+#: transit_indicators/models.py:421
 msgid "Ratio of transit lines length over road length"
 msgstr "Tỉ số chiều dài chặng trên độ dài đường phố"
 
-#: transit_indicators/models.py:406
+#: transit_indicators/models.py:422
 msgid "Transit line network density"
 msgstr "Mật độ mạng lưới đường giao thông"
 
-#: transit_indicators/models.py:407
+#: transit_indicators/models.py:423
 msgid "Number of modes"
 msgstr "Số lượng các chế độ"
 
-#: transit_indicators/models.py:408
+#: transit_indicators/models.py:424
 msgid "Number of routes"
 msgstr "Số lượng tuyến"
 
-#: transit_indicators/models.py:409
+#: transit_indicators/models.py:425
 msgid "Number of stops"
 msgstr "Số lượng điểm dừng"
 
-#: transit_indicators/models.py:410
+#: transit_indicators/models.py:426
 msgid "Number of route types"
 msgstr "Số lượng kiểu tuyến"
 
-#: transit_indicators/models.py:411
+#: transit_indicators/models.py:427
 msgid "On-Time Performance"
 msgstr "Thực hiện đúng giờ"
 
-#: transit_indicators/models.py:412
+#: transit_indicators/models.py:428
 msgid "Regularity of Headways"
 msgstr "Mức phổ biến của điểm đầu tuyến"
 
-#: transit_indicators/models.py:413
+#: transit_indicators/models.py:429
 msgid "Service frequency weighted by served population"
 msgstr "Tần số dịch vụ xác định theo dân số phục vụ"
 
-#: transit_indicators/models.py:414
+#: transit_indicators/models.py:430
 #, fuzzy
 #| msgid "Service frequency weighted by served population"
-msgid "Service frequency weighted by served low-income population"
+msgid "Service frequency weighted by population metric 2"
 msgstr "Tần số dịch vụ xác định theo dân số phục vụ"
 
-#: transit_indicators/models.py:415
+#: transit_indicators/models.py:431
 msgid "Ratio of number of stops to route-length"
 msgstr "Tỉ số số lượng điểm dừng trên chiều dài tuyến"
 
-#: transit_indicators/models.py:416
+#: transit_indicators/models.py:432
 msgid "Ratio of the Transit-Pattern Operating Suburban Lines"
 msgstr "Tỉ lệ mạng lưới các đường ở khu vực ngoại ô"
 
-#: transit_indicators/models.py:417
+#: transit_indicators/models.py:433
 msgid "System accessibility"
 msgstr "Khả năng đi đến của hệ thống"
 
-#: transit_indicators/models.py:418
-msgid "System accessibility - low-income"
+#: transit_indicators/models.py:434
+#, fuzzy
+#| msgid "System accessibility - low-income"
+msgid "System accessibility - population metric 2"
 msgstr "Khả năng đi đến của hệ thống – thu nhập thấp"
```

Comment:
I don't know whether the translators use a program like POEdit, so it might be best simply to delete msgstr (and the `fuzzy` keyword) to make it clear that they should be re-translated.